cli/test/test_parser.py

```diff
@@ -1074,7 +1074,6 @@ def test_normalize(self):
         hcl_workflow.normalize()
         self.assertEqual(hcl_workflow.resolves, ['a'])
         self.assertEqual(hcl_workflow.name, 'sample workflow')
-        self.assertEqual(hcl_workflow.on, 'push')
```

Comment:
all tests in this file can also be left as is

Reply:
in that case, won't these tests show errors if run?